package.json

```diff
@@ -301,12 +301,12 @@
       {
         "command": "one.explorer.deleteOnShortcut",
         "key": "delete",
-        "when": "OneExplorerView.active"
+        "when": "view.OneExplorerView.visible && !explorerResourceReadonly && !inputFocus"
```

Comment:
@dayo09 Where is `explorerResourceReadonly` comes from?

Reply:
You can refer [this url](https://www.whitphx.info/posts/20230217-vscode-undocumented-contexts/) about undocumented contexts. :(

Reply:
@hseok-oh Actually, I thought it's a flag whether the selected node's uri file is readonly or not, but it was always true for readonly file both on default explorer and one explorer...